kroxylicious-filter-archetype/src/main/resources/archetype-resources/src/main/java/SampleFetchResponse.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
```

Comment:
should we drop all our license bits from these templates, feels a bit wrong to lay claim to archetype generated modules.

Reply:
The intent of the archetype is to give users a starting point for their coding.  Licensing the generating files will be an impediment to organisation that require their own headers (copyrights etc) files on their sources.  I think we remove it.